lib/tasks/creatorships.rake

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+namespace :creatorships do
+  desc "Clean up creatorships for deleted chapters"
```

Comment:
I added clean-up tasks for series and chapters because they currently don't have `dependent: :destroy` set on their creatorships, while works do. So while I'm positive that there are plenty of lingering creatorships for deleted chapters and deleted series, I suspect that there are relatively few lingering creatorships for deleted works.

But I guess I'll add the task regardless. It doesn't hurt to add it, and it might be helpful if there are more issues with works than I suspected.

Reply:
I just checked:
```
2.3.4 :003 > Creatorship.joins("LEFT JOIN works ON creatorships.creation_id = works.id AND creatorships.creation_type = \"Work\"").where(works: { id: nil }).count
 => 18482298
```
We'll need the task.